.github/workflows/ci.yml

```diff
@@ -0,0 +1,29 @@
+name: prolog / main
+
+on:
+  push:
+    branches: [master, main]
+  pull_request:
+  workflow_dispatch:
+
+jobs:
+  ci:
+    runs-on: ubuntu-latest
+
+    strategy:
+      fail-fast: false
+      matrix:
+        prolog-branch: [stable, devel]
```

Comment:
I noticed that it runs on the develop channel, which seemed to be case with Travis too. Interesting!

Reply:
Yeah, for completion's sake, I added `stable` to the matrix here, but it was only using `devel` on Travis.